### ISSUE TITLE
Changing src directory in conf.js breaks dist build target

### ISSUE THREAD

**Issue by skjolber** [Swiip/generator-gulp-angular#941]
This makes adding the webapp difficult for a maven build

**Comment by busches** [comment]
An example of what you changed and the error would help to fix this.

**Comment by skjolber** [comment]
mkdir test, cd test, yo gulp-angular with all default options. rename src folder to abc, modify gulp/conf.js file line 15 correspondingly. gulp serve works. gulp serve:dist fails with message '
Error: [$injector:nomod] Module 'test' is not available!'

**Comment by dfromero** [comment]
@skjolber  you have to rename as well on the index.html, i think i related to an issue i had #932

**Comment by Swiip** [comment]
Yep, it's not enough to only change paths in the conf.js. We should have a recipe for this...
